### PR TITLE
readcd: add page

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
Lazy starting point for this command. There's also `cdrecord` but I have no idea how that functions.

File: pages/linux/readcd.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# readcd
+
+> Read or write Compact Disc media data.
+> More information: <https://manned.org/readcd>.
+
+- Read a cd and copy it to a file:
+
+`readcd dev={{/dev/sr0}} f=path/to/file.iso`
```
Comment (spageektti) [inline pages/linux/readcd.md:8]:
```suggestion
`readcd dev={{/dev/sr0}} f={{path/to/file.iso}}`
```

**Review by gutjuri — APPROVED** [review]
LGTM after the suggestion

File: pages/linux/readcd.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# readcd
+
+> Read or write Compact Disc media data.
+> More information: <https://manned.org/readcd>.
+
+- Read a cd and copy it to a file:
+
+`readcd dev={{/dev/sr0}} f={{path/to/file.iso}}`
```
Comment (spageektti) [inline pages/linux/readcd.md:8]:
```suggestion
`readcd dev={{/dev/srX}} f={{path/to/file.iso}}`
```

**Review by spageektti — APPROVED** [review]

**Pull request merged**